src/lib/components/pane/code/editor.svelte

```diff
@@ -140,6 +141,19 @@
             }),
             parent,
         });
+        zoomSize.subscribe((size) => {
+            view!.contentDOM.style.fontSize = `${size}px`;
+            for (const num of parent.querySelectorAll('.cm-gutterElement')) {
+                (num as HTMLElement).style.fontSize = view!.contentDOM.style.fontSize;
+            }
+        });
+        parent.addEventListener('wheel', (e) => {
+            if (e.ctrlKey) {
+                e.preventDefault();
+                const toAdd = e.deltaY > 0 ? -1 : 1;
+                zoomSize.set(Math.max(8, Math.min(24, $zoomSize + toAdd)));
```

Comment:
I think the shorthand can be used here
```suggestion
                $zoomSize = Math.max(8, Math.min(24, $editorTextSize + toAdd));
```